base_export_manager/static/src/js/base_export_manager.js

```diff
@@ -23,5 +31,60 @@ odoo.define('base_export_manager.base_export_manager', function(require) {
             }
         },
     });
-    
+
+
+    Session.include({
+        get_export_models: function() {
+            if (!this.uid) {
+                return $.when().resolve(false);
+            }
+            var Model = window.openerp.web.Model;
+            var Users = new Model('res.users');
+            var export_models = Users.call('get_export_models', []);
+            return export_models;
+        },
+    });
+
+    ListView.include({
+        view_loading: function(fvg) {
+            this._super(fvg);
+            this.is_export_manager();
+        },
+        is_export_manager: function () {
+            var self = this;
+            $.when(self.session.get_export_models()).then(function(export_models) {
+                self.export_models=export_models;
+            });
+        },
+        /**
+         * Instantiate and render the sidebar.
+         * Sets this.sidebar
+         * @param {jQuery} [$node] a jQuery node where the sidebar should be inserted
+         * $node may be undefined, in which case the ListView inserts the sidebar in
+         * this.options.$sidebar or in a div of its template
+         **/
+        render_sidebar: function($node) {
+            var self = this;
+            self.render_export_enable = jQuery.inArray( this.model, self.export_models );
+            if (!this.sidebar && this.options.sidebar) {
+                this.sidebar = new Sidebar(this, {editable: this.is_action_enabled('edit')});
+                if (this.fields_view.toolbar) {
+                    this.sidebar.add_toolbar(this.fields_view.toolbar);
+                }
+                this.sidebar.add_items('other', _.compact([
+                    self.render_export_enable >= 0 && { label: _t("Export"), callback: this.on_sidebar_export },
```

Comment:
Can't this be done in the Qweb template instead, or the render is all in JS code?